addons/resource/models/resource_calendar.py

```diff
@@ -191,6 +220,18 @@ def _get_hours_per_day(self, attendances):
 
         return float_round(hour_count / float(number_of_days), precision_digits=2)
 
+    def _get_days_per_week(self):
+        # Returns the number of days per week during which the employee is working
+        # For examples
+        # 38h / weeks -> 5 days
+        # 19h / weeks (M/T/Wam) -> 3 days
+        # 19h / weeks (Mam/Tam/Wam/Tam/Fam) -> 5 days
+        self.ensure_one()
+        if self.two_weeks_calendar:
+            return 5 * self.work_time_rate / 100
+        return len(set(self.attendance_ids.filtered(
+            lambda a: a._is_work_period()).mapped('dayofweek')))
```

Comment:
i think our previous computaion was false, we should ask geoffrey about it :)

because 5 is really random (for example in France you can work 6 days per week not only 5 :/)